staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/mockarmclient/interface.go

```diff
@@ -1,4 +1,3 @@
-//go:build !providerless
```

Comment:
@tssurya this change is unrelated and makes the job fail, it seems you only need
```
pkg/generated/openapi/zz_generated.openapi.go
```

Reply:
ahh yeah probably oops :)

Reply:
done.